crates/node/src/lib.rs

```diff
@@ -77,6 +77,19 @@ pub async fn start(config: NodeConfig) -> eyre::Result<()> {
 
     let mut stdin = tokio::io::BufReader::new(tokio::io::stdin()).lines();
 
+    match network_client
+        .subscribe(IdentTopic::new(
```

Comment:
Is this just a workaround or do we plan to leave it like this?

Reply:
For now, we don't have a different solution in the plan.